lib/analysis/modules/rml_enkf.c

```diff
@@ -620,6 +620,7 @@ void rml_enkf_updateA(void * module_data,
 
 void rml_enkf_init_update(void * arg,
                           const bool_vector_type * ens_mask,
+                          const bool_vector_type * obs_mask,
```

Comment:
I guess I can read this change as this code actually having zero tests?

Reply:
Not even compiled by default ... :-(

Reply:
Scary stuff; robots from the future will come for us soon!